js/client-mainmenu.js

```diff
@@ -295,6 +296,7 @@
 							$chat.append('<div class="chat">User ' + userid + ' no longer ignored.</div>');
 						}
 					} else {
+						this.chatHistories[userid].push(text);
```

Comment:
I actually added line 299 before I knew that line 282 existed, and then removed line 282 as part of a cleanup sweep without realising what I'd done. I've moved it back (so to speak).